File: June Challenge/Week 1/Java/Day_4/ReverseString.java
//Problem:https://leetcode.com/problems/reverse-string/
//Solution:
class Solution {
    public void reverseString(char[] s) {
        char a;
        int left=0;
        int right=s.length-1;
        while(left<=right){
            a=s[left];
            s[left]=s[right];
            s[right]=a;
            left++;
            right--;
        }
    }
}

File: June Challenge/Week 1/C++/DAY_4/ReverseString.cpp
void swap(char &c1 , char &c2){
    char c3=c1;
    c1=c2;
    c2=c3;
}

class Solution {
public:
    void reverseString(vector<char>& s) {
        for(int i=0;i<s.size()/2;i++){
            swap(s.at(i),s.at(s.size()-i-1));
        }
    }
};
